drivers/storage/portworx/portworx.go

```diff
@@ -149,6 +149,11 @@ func (p *portworx) Validate(cluster *corev1.StorageCluster) error {
 		}
 	}
 
+	if !pxutil.IsEKS(cluster) {
+		logrus.Warnf("pre-flight is not enabled on this environment")
+		return nil
+	}
+	
```

Comment:
I am not so sure this is the right place to do this?  We have a function which determines if pre-flight is supposed to run:  https://github.com/libopenstorage/operator/blob/35b110c2aa33a3f0773ec6e4e75e30d3be835238/drivers/storage/portworx/portworx.go#L477.  I think its better to disable there in master.  

But in 12.10.2 branch its a bit different portworx.go:preflightShouldRun() at line ~295.